### ISSUE TITLE
Divine Shield error in Battlegrounds

### ISSUE THREAD

**Issue by 0xZeb** [RetroWoW/RetroWoW#512]
**Description**: Paladin not immune to all spells in Divine Shield. 

**Current behavior**: Paladin with buff Divine Shield not Immune to Warlock pet spell Seduction.

**Expected behavior**: Paladin with Divine Shield buff should be immune to Warlock pet spell Seduction.

**Steps to reproduce the problem**:

1. Cast divine shield 
2. Have warlock Cast Seduction
3. Be seduced in bubble, immune from other spells, but unable to cast or move.
4. Happens at Gold Mine in Arathi Basin.... hard to replicate in Hyjal or  in duels.

**Source:** I swear I have been seduced inside divine shield several times in battlegrounds. I ill update with screens when I can get them. 

**Include Screenshots from the issue if necessary**:

**Comment by 0xZeb** [comment]
Read combat log on top pic. Divine Shield, then Seduction gained.
![test_bug](https://user-images.githubusercontent.com/35232687/86461825-6ebf9680-bcdf-11ea-927c-1441fa1df47f.png)

![debuffs_buffs_seduce_divineshield](https://user-images.githubusercontent.com/35232687/86461700-3c159e00-bcdf-11ea-91ca-41aff2ff1e05.png)

**Comment by VovremyaRetroWoW** [comment]
Yep, as I expected, it should be possible only while casting of seduction and that's not OK. If you see casting of seduction, it's already too late to bubble it. :)
